.github/workflows/libkineto_cuda.yml

```diff
@@ -0,0 +1,72 @@
+name: libkineto PR Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+# Use TorchBench's docker image which has all basic dependencies.
+env:
+  CONDA_ENV: "torchbench"
+  DOCKER_IMAGE: "ghcr.io/pytorch/torchbench:latest"
+  SETUP_SCRIPT: "/workspace/setup_instance.sh"
+
+jobs:
+  pr-test:
+    # AWS linux instance label: c5.2xlarge
+    runs-on: [linux.2xlarge]
+    timeout-minutes: 180 # 3 hours
+    steps:
+      - name: Checkout Kineto
+        uses: actions/checkout@v3
+        with:
+          path: kineto
+      - name: Checkout submodules
+        shell: bash
+        run: |
+          auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+          git submodule sync --recursive
```

Comment:
I guess you need to `cd kineto` first before running `git submodule ...` commands